### ISSUE TITLE
ERROR {migration.service.SchemaMigrator} - Error occurred while executing SQL script for migrating : CALL ALTER_IDN_OAUTH2_DEVICE_FLOW (5.11.0 Latest U2 to 5.12.0 Beta3 | DB2 11.5)

### ISSUE THREAD

**Issue by ShanikaWickramasinghe** [wso2/product-is#13698]
**How to reproduce:**

1.Get IS 5.11.0 GA pack from website and get it updated to latest u2 level 145
2. Get IS 5.12.0 beta 3 pack
3. Configure primary userstore for unique id jdbc US and db type as DB2 11.5
4. Get the migration client version v1.0.183
5. Run the migration client (Note:- Sample data such as SP(oidc,saml), IDP (fb), scope, roles, groups, users, secondory US, tenant, sample apps users and roles registered under a new tenant were created in the old IS pack before the migration.

5.12.0 beta 3

```
[server]
hostname = "localhost"
node_ip = "127.0.0.1"
base_path = "https://$ref{server.hostname}:${carbon.management.port}"
offset = "1"

[super_admin]
username = "admin"
password = "admin"
create_admin_account = true

[user_store]
type = "database_unique_id"

[database.identity_db]

url = "jdbc:db2://localhost:50000/twlmig2"
username = "db2inst1"
password = "wso2carbon"
driver = "com.ibm.db2.jcc.DB2Driver"
[database.identity_db.pool_options]
maxActive = "80"
minIdle ="5"
testOnBorrow = true
validationQuery="SELECT 1 FROM SYSIBM.SYSDUMMY1"
validationInterval="30000"
defaultAutoCommit=false

[database.shared_db]

url = "jdbc:db2://localhost:50000/twlmig2"
username = "db2inst1"
password = "wso2carbon"
driver = "com.ibm.db2.jcc.DB2Driver"
[database.shared_db.pool_options]
maxActive = "80"
minIdle ="5"
testOnBorrow = true
validationQuery="SELECT 1 FROM SYSIBM.SYSDUMMY1"
validationInterval="30000"
defaultAutoCommit=false

[keystore.primary]
file_name = "wso2carbon.jks"
password = "wso2carbon"

[keystore.internal]
file_name = "internal.jks"
type = "JKS"
password = "wso2carbon"
alias = "wso2carbon"
key_password = "wso2carbon"

[truststore]
file_name="client-truststore.jks"
password="wso2carbon"
type="JKS"

[account_recovery.endpoint.auth]
hash= "66cd9688a2ae068244ea01e70f0e230f5623b7fa4cdecb65070a09ec06452262"

[identity.auth_framework.endpoint]
app_password= "dashboard"

[system_applications]
read_only_apps = []

```
Getting below exception while running the migration client

```
[2022-05-17 23:53:41,843] []  INFO {org.wso2.carbon.is.migration.VersionMigration} -  WSO2 Product Migration Service Task : Version : 5.12.0, Migration Step : SchemaMigrator of order : 7 is starting........................... 
[2022-05-17 23:53:41,844] []  INFO {org.wso2.carbon.is.migration.service.SchemaMigrator} -  WSO2 Product Migration Service Task : Executing Identity Migration Scripts.
[2022-05-17 23:53:41,899] []  INFO {org.wso2.carbon.is.migration.service.SchemaMigrator} -  WSO2 Product Migration Service Task : Identity DB Migration script executed successfully.
[2022-05-17 23:53:41,900] []  INFO {org.wso2.carbon.is.migration.VersionMigration} -  WSO2 Product Migration Service Task : Version : 5.12.0, Migration Step : SchemaMigrator of order : 8 is starting........................... 
[2022-05-17 23:53:41,900] []  INFO {org.wso2.carbon.is.migration.service.SchemaMigrator} -  WSO2 Product Migration Service Task : Executing Identity Migration Scripts.
[2022-05-17 23:53:41,915] []  INFO {org.wso2.carbon.is.migration.service.SchemaMigrator} -  WSO2 Product Migration Service Task : Identity DB Migration script executed successfully.
[2022-05-17 23:53:41,916] []  INFO {org.wso2.carbon.is.migration.VersionMigration} -  WSO2 Product Migration Service Task : Version : 5.12.0, Migration Step : SchemaMigrator of order : 9 is starting........................... 
[2022-05-17 23:53:41,916] []  INFO {org.wso2.carbon.is.migration.service.SchemaMigrator} -  WSO2 Product Migration Service Task : Executing Identity Migration Scripts.
[2022-05-17 23:53:41,967] [] ERROR {org.wso2.carbon.is.migration.service.SchemaMigrator} - Error occurred while executing SQL script for migrating database java.lang.Exception: Error occurred while executing :   CALL ALTER_IDN_OAUTH2_DEVICE_FLOW 
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQL(SchemaMigrator.java:318)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQLScript(SchemaMigrator.java:234)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.migrate(SchemaMigrator.java:89)
	at org.wso2.carbon.is.migration.VersionMigration.migrate(VersionMigration.java:52)
	at org.wso2.carbon.is.migration.MigrationClientImpl.execute(MigrationClientImpl.java:76)
	at org.wso2.carbon.identity.core.internal.IdentityCoreServiceComponent.activate(IdentityCoreServiceComponent.java:151)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.internal.ds.model.ServiceComponent.activate(ServiceComponent.java:260)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.activate(ServiceComponentProp.java:146)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:345)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponents(InstanceProcess.java:197)
	at org.eclipse.equinox.internal.ds.Resolver.getEligible(Resolver.java:343)
	at org.eclipse.equinox.internal.ds.SCRManager.serviceChanged(SCRManager.java:222)
	at org.eclipse.osgi.internal.serviceregistry.FilteredServiceListener.serviceChanged(FilteredServiceListener.java:113)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:985)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEventPrivileged(ServiceRegistry.java:866)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEvent(ServiceRegistry.java:804)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.register(ServiceRegistrationImpl.java:130)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.registerService(ServiceRegistry.java:228)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:525)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:544)
	at org.wso2.carbon.core.init.CarbonServerManager.initializeCarbon(CarbonServerManager.java:529)
	at org.wso2.carbon.core.init.CarbonServerManager.removePendingItem(CarbonServerManager.java:305)
	at org.wso2.carbon.core.init.PreAxis2ConfigItemListener.bundleChanged(PreAxis2ConfigItemListener.java:118)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:973)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
Caused by: com.ibm.db2.jcc.am.SqlSyntaxErrorException: DB2 SQL Error: SQLCODE=-204, SQLSTATE=42704, SQLERRMC=SQL220517152827740, DRIVER=4.25.13
	at com.ibm.db2.jcc.am.b6.a(b6.java:810)
	at com.ibm.db2.jcc.am.b6.a(b6.java:66)
	at com.ibm.db2.jcc.am.b6.a(b6.java:140)
	at com.ibm.db2.jcc.am.k3.b(k3.java:2471)
	at com.ibm.db2.jcc.am.k3.c(k3.java:2452)
	at com.ibm.db2.jcc.t4.ab.n(ab.java:914)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:156)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.av.k(av.java:175)
	at com.ibm.db2.jcc.am.k3.ao(k3.java:2413)
	at com.ibm.db2.jcc.am.k3.a(k3.java:3383)
	at com.ibm.db2.jcc.am.k3.e(k3.java:1131)
	at com.ibm.db2.jcc.am.k3.execute(k3.java:1110)
	at jdk.internal.reflect.GeneratedMethodAccessor73.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:118)
	at com.sun.proxy.$Proxy57.execute(Unknown Source)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQL(SchemaMigrator.java:273)
	... 32 more

[2022-05-17 23:53:41,992] [] ERROR {org.wso2.carbon.is.migration.service.SchemaMigrator} - An error occurred while rolling back transactions. java.lang.Exception: Error occurred while executing SQL script for migrating database
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQLScript(SchemaMigrator.java:244)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.migrate(SchemaMigrator.java:89)
	at org.wso2.carbon.is.migration.VersionMigration.migrate(VersionMigration.java:52)
	at org.wso2.carbon.is.migration.MigrationClientImpl.execute(MigrationClientImpl.java:76)
	at org.wso2.carbon.identity.core.internal.IdentityCoreServiceComponent.activate(IdentityCoreServiceComponent.java:151)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.internal.ds.model.ServiceComponent.activate(ServiceComponent.java:260)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.activate(ServiceComponentProp.java:146)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:345)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponents(InstanceProcess.java:197)
	at org.eclipse.equinox.internal.ds.Resolver.getEligible(Resolver.java:343)
	at org.eclipse.equinox.internal.ds.SCRManager.serviceChanged(SCRManager.java:222)
	at org.eclipse.osgi.internal.serviceregistry.FilteredServiceListener.serviceChanged(FilteredServiceListener.java:113)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:985)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEventPrivileged(ServiceRegistry.java:866)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEvent(ServiceRegistry.java:804)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.register(ServiceRegistrationImpl.java:130)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.registerService(ServiceRegistry.java:228)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:525)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:544)
	at org.wso2.carbon.core.init.CarbonServerManager.initializeCarbon(CarbonServerManager.java:529)
	at org.wso2.carbon.core.init.CarbonServerManager.removePendingItem(CarbonServerManager.java:305)
	at org.wso2.carbon.core.init.PreAxis2ConfigItemListener.bundleChanged(PreAxis2ConfigItemListener.java:118)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:973)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
Caused by: java.lang.Exception: Error occurred while executing :   CALL ALTER_IDN_OAUTH2_DEVICE_FLOW 
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQL(SchemaMigrator.java:318)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQLScript(SchemaMigrator.java:234)
	... 31 more
Caused by: com.ibm.db2.jcc.am.SqlSyntaxErrorException: DB2 SQL Error: SQLCODE=-204, SQLSTATE=42704, SQLERRMC=SQL220517152827740, DRIVER=4.25.13
	at com.ibm.db2.jcc.am.b6.a(b6.java:810)
	at com.ibm.db2.jcc.am.b6.a(b6.java:66)
	at com.ibm.db2.jcc.am.b6.a(b6.java:140)
	at com.ibm.db2.jcc.am.k3.b(k3.java:2471)
	at com.ibm.db2.jcc.am.k3.c(k3.java:2452)
	at com.ibm.db2.jcc.t4.ab.n(ab.java:914)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:156)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.av.k(av.java:175)
	at com.ibm.db2.jcc.am.k3.ao(k3.java:2413)
	at com.ibm.db2.jcc.am.k3.a(k3.java:3383)
	at com.ibm.db2.jcc.am.k3.e(k3.java:1131)
	at com.ibm.db2.jcc.am.k3.execute(k3.java:1110)
	at jdk.internal.reflect.GeneratedMethodAccessor73.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:118)
	at com.sun.proxy.$Proxy57.execute(Unknown Source)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQL(SchemaMigrator.java:273)
	... 32 more

[2022-05-17 23:53:41,995] [] ERROR {org.wso2.carbon.is.migration.MigrationClientImpl} - Migration process was stopped. org.wso2.carbon.identity.core.migrate.MigrationClientException: Error occurred while executing SQL script for migrating database
	at org.wso2.carbon.is.migration.service.SchemaMigrator.migrate(SchemaMigrator.java:100)
	at org.wso2.carbon.is.migration.VersionMigration.migrate(VersionMigration.java:52)
	at org.wso2.carbon.is.migration.MigrationClientImpl.execute(MigrationClientImpl.java:76)
	at org.wso2.carbon.identity.core.internal.IdentityCoreServiceComponent.activate(IdentityCoreServiceComponent.java:151)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.internal.ds.model.ServiceComponent.activate(ServiceComponent.java:260)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.activate(ServiceComponentProp.java:146)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:345)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponents(InstanceProcess.java:197)
	at org.eclipse.equinox.internal.ds.Resolver.getEligible(Resolver.java:343)
	at org.eclipse.equinox.internal.ds.SCRManager.serviceChanged(SCRManager.java:222)
	at org.eclipse.osgi.internal.serviceregistry.FilteredServiceListener.serviceChanged(FilteredServiceListener.java:113)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:985)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEventPrivileged(ServiceRegistry.java:866)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEvent(ServiceRegistry.java:804)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.register(ServiceRegistrationImpl.java:130)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.registerService(ServiceRegistry.java:228)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:525)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:544)
	at org.wso2.carbon.core.init.CarbonServerManager.initializeCarbon(CarbonServerManager.java:529)
	at org.wso2.carbon.core.init.CarbonServerManager.removePendingItem(CarbonServerManager.java:305)
	at org.wso2.carbon.core.init.PreAxis2ConfigItemListener.bundleChanged(PreAxis2ConfigItemListener.java:118)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:973)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
Caused by: java.lang.Exception: Error occurred while executing SQL script for migrating database
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQLScript(SchemaMigrator.java:244)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.migrate(SchemaMigrator.java:89)
	... 30 more
Caused by: java.lang.Exception: Error occurred while executing :   CALL ALTER_IDN_OAUTH2_DEVICE_FLOW 
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQL(SchemaMigrator.java:318)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQLScript(SchemaMigrator.java:234)
	... 31 more
Caused by: com.ibm.db2.jcc.am.SqlSyntaxErrorException: DB2 SQL Error: SQLCODE=-204, SQLSTATE=42704, SQLERRMC=SQL220517152827740, DRIVER=4.25.13
	at com.ibm.db2.jcc.am.b6.a(b6.java:810)
	at com.ibm.db2.jcc.am.b6.a(b6.java:66)
	at com.ibm.db2.jcc.am.b6.a(b6.java:140)
	at com.ibm.db2.jcc.am.k3.b(k3.java:2471)
	at com.ibm.db2.jcc.am.k3.c(k3.java:2452)
	at com.ibm.db2.jcc.t4.ab.n(ab.java:914)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:156)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.av.k(av.java:175)
	at com.ibm.db2.jcc.am.k3.ao(k3.java:2413)
	at com.ibm.db2.jcc.am.k3.a(k3.java:3383)
	at com.ibm.db2.jcc.am.k3.e(k3.java:1131)
	at com.ibm.db2.jcc.am.k3.execute(k3.java:1110)
	at jdk.internal.reflect.GeneratedMethodAccessor73.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:118)
	at com.sun.proxy.$Proxy57.execute(Unknown Source)
	at org.wso2.carbon.is.migration.service.SchemaMigrator.executeSQL(SchemaMigrator.java:273)
	... 32 more

[2022-05-17 23:53:41,998] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,998] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,999] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,999] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,999] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,999] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,999] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:41,999] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:42,000] []  INFO {org.wso2.carbon.is.migration.MigrationClientImpl} - ............................................................................................
[2022-05-17 23:53:42,445] []  INFO {org.wso2.balana.ConfigurationStore} - Loading runtime configuration
[2022-05-17 23:53:42,451] []  INFO {org.wso2.carbon.identity.entitlement.internal.SchemaBuilder} - XACML policy schema loaded successfully.
[2022-05-17 23:53:42,613] []  INFO {org.wso2.carbon.identity.oauth.uma.xacml.service.internal.AppUMAServiceComponents} - XACML UMA Handler activated successfully.
[2022-05-17 23:53:42,676] []  INFO {org.wso2.carbon.user.core.internal.UserStoreMgtDSComponent} - Claim manager set for class org.wso2.carbon.user.core.common.DefaultRealm
[2022-05-17 23:53:42,679] []  INFO {org.wso2.carbon.user.core.internal.UserStoreMgtDSComponent} - Claim manager set for class org.wso2.carbon.user.core.jdbc.UniqueIDJDBCUserStoreManager
[2022-05-17 23:53:43,479] []  INFO {org.wso2.carbon.identity.local.auth.api.core.internal.AuthAPIServiceComponent} - Auth API Service Component  is activated.
[2022-05-17 23:53:43,634] []  INFO {org.wso2.carbon.identity.oauth.config.OAuthServerConfiguration} - The default OAuth token issuer will be used. No custom token generator is set.
[2022-05-17 23:53:43,634] []  INFO {org.wso2.carbon.identity.oauth.config.OAuthServerConfiguration} - An instance of org.wso2.carbon.identity.oauth2.token.OauthTokenIssuerImpl is created for Identity OAuth token generation.
[2022-05-17 23:53:43,648] []  INFO {org.opensaml.core.config.InitializationService} - Initializing OpenSAML using the Java Services API
[2022-05-17 23:53:44,553] []  WARN {org.wso2.carbon.identity.conditional.auth.functions.common.utils.ConfigProvider} - Choreo domain list used by the callChore function is not configured therefore domain restriction is turned off for callChoreo function.
[2022-05-17 23:53:44,786] []  INFO {org.wso2.carbon.identity.application.authentication.framework.store.SessionDataStore} - Thread pool size for temporary authentication context data delete task: 20
[2022-05-17 23:53:45,354] []  INFO {org.wso2.carbon.identity.workflow.impl.internal.WorkflowImplServiceComponent} - Default BPS profile updated.

```

Attaching carbon logs
[wso2carbon.log](https://github.com/wso2/product-is/files/8710680/wso2carbon.log)

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 IS 5.11.0 latest U2 (level 145) to IS 5.12.0 beta 3 migration
DB2 11.5

**Comment by ashendes** [comment]
I am not able to reproduce this with the latest U2 updated 5.11.0 pack and latest migration client (184) in IS 6.0.0.

**Comment by ShanikaWickramasinghe** [comment]
@ashendes The  configuration steps I have mentioned in the steps to reproduce section is not missing any steps that I have followed. Hope you have created some data under tenant as well before the migration and did the require pre requistite to migrate tenant data. 

The only difference I see is your setup is on latest 5.11.0 U2 update pack level and the migration client version is on 184. ( When I was testing migration client was 183 and 5.11.0 u2 level 145) 

1. What is the U2 level for 5.11.0 in your setup ? 
2. Also I have faced this issue with DB2 driver version DRIVER=4.25.13 which comes with db2jcc-db2jcc4. https://www.ibm.com/support/pages/node/382667. What is the DB2 driver version in your setup.
